test/executor.test.js

```diff
@@ -754,6 +755,26 @@ describe('executor', function() {
       .get('/')
       .expect('passport', 'initialized', done);
   });
+
+  describe('when booting with env', function() {
+    before(function() {
+      process.env.NODE_ENV = 'test';
+    });
+    after(function() {
+      process.env.NODE_ENV = undefined;
+    });
```

Comment:
Please don't manipulate the environment, pass the desired `env` value as an argument of `envAppInstructions`.